modules/special-rules/pages/pulsar-x.adoc

```diff
@@ -1,18 +1,19 @@
-= Pulsar
+= Pulsar (X)
 
 This weapon is unique to the Eldar race, and it demonstrates their mastery of laser weaponry.
 
-The weapon shoots in rapid but powerful bursts, guided by advanced targetting technologies.
+The weapon shoots in rapid but powerful bursts, guided by advanced targeting technologies.
 As a result, the weapon can destroy multiple targets in a single salvo.
 
 ---
 
 When you shoot with a {pulsar}:
 
-* Roll a D6 and resolve this many {macro-weapon} shots against the target detachment.
+. Roll the dice listed in brackets, for example (D3+1) or (D6).
+. Resolve that many {macro-weapon} shots against the target detachment.
```

Comment:
'Dice' is plural, and thus far we expect only a single, hence it should be 'die'. However, for a couple of small reasons, I think it'd be better to rephrase without reference to die/dice anyway. So, how about this:

```suggestion
. Determine the number of shots, based on the value that we state in brackets — for example, `(D3+1)` or `(D6)`.
. Resolve that many {macro-weapon} shots against the target detachment.
```

Reply:
As much as I would love to use 'die/dice', there are enough dictionaries that now list 'dice' as both singular and plural that I think we've lost that fight, as GW have been using 'dice' for both since the Eighties. 🙁

But as you say, it'd be better not to use it anyway.

'the value we state' reads a bit oddly, given that the value will be read from a data sheet. Maybe this?

```suggestion
. Determine the number of shots, based on the value on the unit's data sheet — for example, `(D3+1)` or `(D6)`.
. Resolve that many {macro-weapon} shots against the target detachment.
```

Reply:
I take your point that the incorrect usage is becoming normalised. Here though, we have the luxury that we have full control over the project, and I prefer to take advantage of that to do it right 😁 Indeed, GW has been getting this consistently wrong for a long time — and it still bugs me every time I see it! 😂

> 'the value we state' reads a bit oddly, given that the value will be read from a data sheet.

In this context 'state' clearly means 'in print', not 'out loud' — that's not uncommon in my experience, and seems perfectly okay to me. (Similarly, a 'statement' is often printed content that people read, not something that they hear someone actually vocalise.)

We do also use 'that we state' in similar fashion in the _Transport (X)_ ability text. Alternatively, we could use 'that we specify'.

Yes, we mainly state the value on the unit's data sheet (in brackets, as part of the _Pulsar_ ability keyword, in the 'Notes' column...) — but in principle we might also state it elsewhere (in a paragraph that describes some unique special effect in a scenario, for example). However, I accept that this is very much a lesser consideration.

With some further tweaks, perhaps something like this?

```suggestion
. Determine the number of shots, based on the value of `(X)` that we specify for the ability on the unit's data sheet — for example, `{pulsar-d3plus1}` or `{pulsar-d6}`.
. Resolve that many {macro-weapon} shots against the target detachment.
```

Reply:
I have no preference between 'we state' and 'we specify'.

But I think your first suggestion at https://github.com/GameBrains/er-core/pull/43#discussion_r1176229870 is more concise.

Reply:
> But I think your first suggestion at [#43 (comment)](https://github.com/GameBrains/er-core/pull/43#discussion_r1176229870) is more concise.

Indeed it is — I just wasn't sure how strongly you felt about the "unit's data sheet" part of your last suggestion, but I'm happier to drop that compromise and be more concise.

I think that first suggestion would suffice — but arguably, part of my last suggestion improves on it slightly because it makes direct reference to the 'X' placeholder that exists in the topic/ability title (rather than the slightly indirect "value in brackets"):

```suggestion
. Determine the number of shots, based on the value of `(X)` that we specify — for example, _`(D3+1)`_ or _`(D6)`_.
. Resolve that many {macro-weapon} shots against the target detachment.
```
I think this is fine so I'll apply it shortly — you're always free to raise a new suggestion if you think that I've unwittingly created a new issue in this iteration that I didn't think of.